dash/long_callback/_proxy_set_props.py

```diff
@@ -0,0 +1,7 @@
+class ProxySetProps(dict):
```

Comment:
What's the purpose of `ProxySetProps`? Maybe add a comment at the top of the file.

Reply:
Yes, the app is updated async like with the set_progress, it should be possible to update the same prop as the output.